lib/elastic_settings.rb

```diff
@@ -4,6 +4,29 @@ module ElasticSettings
   KEYWORD = { type: 'keyword',
               normalizer: 'keyword_normalizer' }.freeze
 
+  TEXT = {
+           properties: {
+             en: {
+               type: 'text',
+               analyzer: 'en_analyzer',
+               copy_to: 'bigram',
+               term_vector: 'with_positions_offsets'
+             },
+             es: {
+               type: 'text',
+               analyzer: 'es_analyzer',
+               copy_to: 'bigram',
+               term_vector: 'with_positions_offsets'
+             },
+             babel: {
+               type: 'text',
+               analyzer: 'babel_analyzer',
+               copy_to: 'bigram',
+               term_vector: 'with_positions_offsets'
+             }
+           }
+         }
+
```

Comment:
RuboCop: Style/MutableConstant: Freeze mutable objects assigned to constants. [Style/MutableConstant
RuboCop: Layout/IndentHash: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is. [Layout/IndentHash]

Reply:
fixed